benchmarks/README.md

```diff
@@ -0,0 +1,46 @@
+# Running benchmarks 
+
+We're using [pytest-benchmark](https://pytest-benchmark.readthedocs.io/en/latest/) to assess performance. Benchmarks can be run with 
+
+```
+pytest benchmarks/ --cutest
```

Comment:
Ah, looks like this needs syncing with `CONTRIBUTING.md`.

What happens if you only run `pytest benchmarks`?

What stops the benchmarks from being ran on just `pytest`?

How do you want to handle forward compatibility with any of our own benchmarks in the future? I'm anticipating that the `--cutest` flag would then no longer be the right name as we probably do want people to run our own benchmarks at the same time, and not make the distinction.